packages/exchange/src/translators/module.ts

```diff
@@ -0,0 +1,39 @@
+import {Exchange, ExchangeProviders} from '@yoroi/types'
+import {handleZodErrors} from '../adapters/zod-errors'
+import {urlReferralQueryStringParamsSchema} from '../adapters/zod-schema'
+import {generateBanxaBaseUrl} from '../adapters/banxa/baseUrl'
+
+export const exchangeModuleMaker = (
+  {partner, isProduction}: Exchange.ReferralUrlBuilderOptions,
```

Comment:
it can be renamed to exchangeManagerMaker -> later it will be handling both APIs, banxa and encryptus.
```suggestion
  {partner, isProduction}: Exchange.ManagerOptions,
```